web/design-tokens.json

```diff
@@ -0,0 +1,5683 @@
+{
```

Comment:
is this file generated from figma?

Reply:
Yes, it is generated with the only plugin that works for dev mode accounts, and it makes me a bit nervous about the longevity and availability of these Figma plugins, so I'm opting to commit everything to ensure we aren't interrupted by anything that happens to change there over time.